uuid.go

```diff
@@ -153,6 +156,28 @@ func (u *UUID) SetVariant(v byte) {
 	}
 }
 
+// UnixNanoOf returns the number of nanoseconds in a V1 UUID
+func (u *UUID) UnixNanoOf() (int64, error) {
+	if u.Version() != 1 {
+		err := fmt.Errorf("uuid: %s is version %d, not version 1", u, u.Version())
+		return 0, err
+	}
+	low := binary.BigEndian.Uint32(u[0:4])
+	mid := binary.BigEndian.Uint16(u[4:6])
+	hi := binary.BigEndian.Uint16(u[6:8]) & 0xfff
+	return 100 * (int64(low) + (int64(mid) << 32) + (int64(hi) << 48) - int64(epochStart)), nil
```

Comment:
When you have a moment would you please add some spacing to this block of code and put the return calculation on a separate line (for readability purposes)? For spacing I'm just thinking of an empty line before the `low` line and after the `hi` line.

For the return, I think it'd be okay to put the calculation on its own line so that we can then have `return var, nil`. For me at least, it made it easier to grok on a quick look.